scripts/setup-k8s-master.sh.in

```diff
@@ -120,13 +140,10 @@ kubectl apply -f {{NetworkPolicyUrl}}
 
 # Use kubeadm's kubeconfig command to grab a client-cert-authenticated
 # kubeconfig file for administrative access to the cluster.
-# Because kubeadm doesn't set controlPlaneEndpoint as master, the
-# --apiserver-advertise-address is set to the load balancer external ipv4 address.
 KUBECONFIG_OUTPUT=/home/ubuntu/kubeconfig
 
-kubeadm alpha phase kubeconfig user \
+kubeadm alpha kubeconfig user \
   --client-name admin \
-  --apiserver-advertise-address "${LB_IPV4}" \
```

Comment:
Given that this was finally removed, the assignment `LB_IPV4=$(dig +short {{LoadBalancerDns}})` can probably be removed too.

Reply:
This was actually put back in as otherwise the kubeconfig was advertising the api server's private address.